### PR TITLE
Remove scroll listener preload script that is no longer used. r=vporof

### PR DESCRIPTION
**Pull request by jsantell**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -33,7 +33,6 @@
     "express-ws": "2.0.0-rc.1",
     "fs-promise": "0.5.0",
     "immutable": "3.7.6",
-    "lodash": "4.13.1",
```
Comment (victorporof) [inline package.json]: This is surprising. We still use lodash in tests though.

File: app/shared/preload/scroll.js [hunk 2]
```diff
@@ -1,24 +0,0 @@
...
-import { ipcRenderer as ipc } from 'electron';
-import throttle from 'lodash/throttle';
-
-// How often the scroll handler may be fired at most
-const THROTTLE_TIMER_IN_MS = 50;
-
-window.addEventListener('scroll', throttle(() => {
-  ipc.sendToHost('scroll', {
-    x: window.scrollX,
-    y: window.scrollY,
```
Comment (victorporof) [inline app/shared/preload/scroll.js]: I can imagine us wanting to use this again at some point. Would personally like to keep it, but it's your choice.

File: package.json (hunk 1 above)
Comment (victorporof) [inline package.json]: I wonder why removing this line didn't make the tests fail. Linting tests use `lodash/without` and `lodash/difference`

Comment (jsantell) [inline package.json]: Huh. Weird. I'll add it back as a devDependency atleast for now

Comment (victorporof) [inline package.json]: See https://github.com/mozilla/tofino/issues/647 
I think node_modules should be clobbered.

**Pull request merged**